pkg/networkservice/common/externalips/server.go

```diff
@@ -64,20 +67,21 @@ func NewServer(chainCtx context.Context, options ...Option) networkservice.Netwo
 	for _, o := range options {
 		o(result)
 	}
+	chainCtxWithLog, log := logruslogger.New(chainCtx)
```

Comment:
Why we cannot put it into chain context in the place where chain context decorated?